src/aap_eda/settings/default.py

```diff
@@ -501,3 +501,18 @@ def _get_secret_key() -> str:
 
 PG_NOTIFY_DSN = settings.get("PG_NOTIFY_DSN", _DEFAULT_PG_NOTIFY_DSN)
 PG_NOTIFY_TEMPLATE_RULEBOOK = settings.get("PG_NOTIFY_TEMPLATE_RULEBOOK", None)
+_DEFAULT_PG_NOTIFY_DSN_SERVER = (
+    f"host={DATABASES['default']['HOST']} "
+    f"port={DATABASES['default']['PORT']} "
+    f"dbname={DATABASES['default']['NAME']} "
+    f"user={DATABASES['default']['USER']} "
+    f"password={DATABASES['default']['PASSWORD']}"
+)
+PG_NOTIFY_DSN_SERVER = settings.get(
+    "PG_NOTIFY_DSN_SERVER", _DEFAULT_PG_NOTIFY_DSN_SERVER
+)
+SERVER_UUID = settings.get("SERVER_UUID", "abc-def-123-34567")
+WEBHOOK_URL_PREFIX = settings.get(
+    "WEBHOOK_URL_PREFIX",
+    f"https://ui.eda.local:8443/{SERVER_UUID}/api/eda/v1/external_webhook",
+)
```

Comment:
Should be these UUID empty by default, specially after the discussion when we agreed on implement it later